pkgs/development/libraries/blitz/patch/0001-fix-configure-script-to-detect-boost-dylib-files.patch

```diff
@@ -0,0 +1,25 @@
+From 6456ee7b0185c1edac327f2504efc3037fc578e2 Mon Sep 17 00:00:00 2001
```

Comment:
No need to patch the configure script.
```diff
diff --git a/pkgs/development/libraries/blitz/default.nix b/pkgs/development/libraries/blitz/default.nix
index 0b06bfd6483..08991d2b98d 100644
--- a/pkgs/development/libraries/blitz/default.nix
+++ b/pkgs/development/libraries/blitz/default.nix
@@ -45,6 +45,9 @@ stdenv.mkDerivation rec {
     ++ optional enableSerialization "--enable-serialization"
     ++ optional stdenv.is64bit "--enable-64bit";
 
+  # skip broken library name detection
+  ax_boost_user_serialization_lib = lib.optionalString stdenv.isDarwin "boost_serialization";
+
   enableParallelBuilding = true;
 
   inherit doCheck;
```

Reply:
Thank you, this is much cleaner. I've made these changes.

Reply:
Great. Please squash the commits.

Reply:
Done, thanks.